app/controllers/public/sessions.js

```diff
@@ -37,9 +37,9 @@ export default class SessionsController extends Controller {
     const arrSet = new Set();
 
     arr.forEach(el => {
-      el = el.split('T')[0];
+      el = moment(el).utc().format('YYYY-MM-DD');
       for (let j = 0; j < arr.length; j++) {
-        if (el === arr[j].split('T')[0]) {
+        if (el === moment(arr[j]).utc().format('YYYY-MM-DD')) {
```

Comment:
Use map and filter

Reply:
sir , map will return a new array with same no of elements , but i don't want to do it here,
what i did , find all the elements of arr with same date and pushed those inside a set and set will automatically make it unique and that's why we are not getting repeated dates in the array.

Reply:
Use Set